### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by makeasnek**
Clarify confusing sentence

### CODE REVIEW

File: wiki/faq.md [hunk 1]
```diff
@@ -642,4 +642,4 @@ have a lot of small transactions.
 7. Right-click the Gridcoin Shortcut, select properties, and remove `-choosedatadir`
 
 
----
\ No newline at end of file
+---
```
Comment (barton2526) [inline wiki/faq.md:645]: what is going on down here?

File: wiki/faq.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ Consult these guides:
 ### I get an "e-mail mismatch" or "split CPID" error in the solo crunching wizard in the wallet?
 First, make sure you are using the same e-mail address for each BOINC project you are attached to.
 
-Some context: CPIDs are IDs assigned to you by BOINC projects based on your e-mail address and publish user stats by CPID (which Gridcoin uses to calculate your rewards). When you first attach to a project, it gives you a CPID (so all projects will have different ones). Your BOINC client then picks the oldest CPID and tells the projects to use it going forward.
+Some context: CPIDs are IDs assigned to you by BOINC projects based on your e-mail address. BONIC projects publish user stats by CPID (which Gridcoin uses to calculate your rewards). When you first attach to a project, it gives you a CPID (so all projects will have different ones). Your BOINC client then picks the oldest CPID and tells the projects to use it going forward.
```
Comment (barton2526) [inline wiki/faq.md:62]: "BONIC"

File: wiki/faq.md (hunk 1 above)
Comment (makeasnek) [reply wiki/faq.md]: Hmm weird I don't remember adding that. Removed.

File: wiki/faq.md (hunk 2 above)
Comment (makeasnek) [reply wiki/faq.md]: Good catch. Fixed

**Review by barton2526 — APPROVED** [review]

**Comment by makeasnek** [comment]
Thanks for the quick merge on this one :)

**Pull request merged**